play-ahc-ws-standalone/src/main/java/play/libs/ws/ahc/AhcWSCookie.java

```diff
@@ -20,31 +20,43 @@ public AhcWSCookie(play.shaded.ahc.org.asynchttpclient.cookie.Cookie ahcCookie)
     /**
      * Returns the underlying "native" object for the cookie.
      */
-    public Object getUnderlying() {
+    @Override
+    public play.shaded.ahc.org.asynchttpclient.cookie.Cookie getUnderlying() {
```

Comment:
I'm going to refactor `WSCookie` in any case as part of https://github.com/playframework/play-ws/issues/97